### PR TITLE
- fixes typo in conditional access page

### PR DESCRIPTION
**Pull request by baywet**

### CODE REVIEW

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/business/set-up-conditional-access-policies.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ Currently, baseline policies include the following:
 - **Block legacy authentication** &ndash; Older client apps and some new apps don't use newer, more secure, authentication protocols. These older apps can bypass conditional access policies and gain unauthorized access to your environment. This policy blocks access from clients that don't support conditional access. 
 - **Require MFA for Service Management** &ndash; Requires multi-factor authentication for access to management tools, including Azure portal (where you configure baseline policies). 
 
-Microsoft recommends you enable all of these baseline policies. After these policies are enabled, admins and users will be prompted to register for Azure Multii-Factor authentication.
+Microsoft recommends you enable all of these baseline policies. After these policies are enabled, admins and users will be prompted to register for Azure Multi-Factor authentication.
```
Comment (JohanFreelancer9) [inline microsoft-365/business/set-up-conditional-access-policies.md]:
```suggestion
We recommend you enable all of these baseline policies. After these policies are enabled, admins and users will be prompted to register for Azure Multi-Factor authentication.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - additional edit; "Microsoft recommends" should not be used.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by mypil** [comment]
@Sirkkuw - PR has been copy edited and is ready for final review. 


Thank you.

cc: @kenwith

**Pull request merged**